hs_core/models.py

```diff
@@ -2939,7 +2939,7 @@ class Meta:
     file_folder = models.CharField(max_length=4096, null=False, default="")
 
     # This pair of FileFields deals with the fact that there are two kinds of storage
-    resource_file = models.FileField(upload_to=get_path, max_length=4096,
+    resource_file = models.FileField(upload_to=get_path, max_length=4096, unique=True,
                                      null=True, blank=True, storage=IrodsStorage())
     fed_resource_file = models.FileField(upload_to=get_path, max_length=4096,
```

Comment:
Should this be set to `unique=True` ?

Reply:
Originally I had `unique=True` for the fed_resource_file as well. The issue is, we have `blank=True` so all of the values are defaulting to an EMPTY string "". Then if we add `unique=True`, it will fail because every "" is considered a non-unique value

Reply:
Since `unique=True` is not causing the same issue for the `resource_resource` field, I am assuming it does not have any record with empty string. May be we need another management command to set any `resource_file/fed_resource_file` from empty string to None. Also should we be finding duplicate files based on `fed_resource_file` and delete those?

Reply:
Thanks @pkdash I did more research and discovered that due to an issue in Django, we can't save a FileField as None/NULL in Django...see https://code.djangoproject.com/ticket/10244

So this is pretty unsatisfying and I bet it's why we didn't have a `unique` constraint on these fields historically.

Since we don't have any federated_resource_file values that aren't ""/EMPTY, I vote that we leave this as is (even though I agree that there should ideally be a unique constraint...I added a comment along these lines)